packages/pieces/apps/src/lib/gmail/triggers/new-email.ts

```diff
@@ -0,0 +1,62 @@
+import { createTrigger, TriggerStrategy } from '@activepieces/framework';
+import dayjs from 'dayjs';
+import { GmailRequests } from '../common/data';
+import { GmailLabel } from '../common/models';
+import { GmailProps } from '../common/props';
+
+export const gmailNewEmailTrigger = createTrigger({
+  name: 'gmail_new_email_received',
+  displayName: 'New Email',
+  description: 'Triggers when new mail is found in your Gmail inbox',
+  props: {
+    authentication: GmailProps.authentication,
+    subject: GmailProps.subject(true),
+    from: GmailProps.from(),
+    to: GmailProps.to(),
+    label: GmailProps.label(),
+    category: GmailProps.category()
+  },
+  sampleData: {
```

Comment:
Ok

Reply:
![image](https://user-images.githubusercontent.com/1812998/223591236-e08c9f3a-762d-43a5-83b9-4b95e407140e.png)

I believe the issue persists as the data has not been enriched and it seems to be returning two emails instead of one. We need to loop through the messages, enrich them, and return an array.